### ISSUE TITLE
why libunwind causes a core dump on the aarch64

### ISSUE THREAD

**Issue by Cwenkang** [libunwind/libunwind#667]
I cross-compiled libunwind 1.7.2（https://github.com/libunwind/libunwind/releases/tag/v1.7.2） on the aarch64 architecture and linked the compiled dynamic library in my project, but as soon as the program runs, the following core dump occurs.

![image](https://github.com/libunwind/libunwind/assets/41962381/e1e53669-38ee-4365-8e8e-3592ef0b15ca)

**Comment by Cwenkang** [comment]
@Gumix  @bregma @am11 @djwatson  Could you please have a look? Thank you!

**Comment by bregma** [comment]
Do you have SVE extensions enabled in your hardware?

Do you have address verification enabled?

**Comment by Cwenkang** [comment]
@bregma
Yes, the Scalable Vector Extension (SVE) is enabled on my hardware, as indicated by sve in the Features line of the /proc/cpuinfo output.

Additionally, Address Space Layout Randomization (ASLR), a form of address verification, is enabled on my system, as indicated by the 2 output from cat /proc/sys/kernel/randomize_va_space.
![image](https://github.com/libunwind/libunwind/assets/41962381/414117ae-cb2c-49d7-bc50-e9514444caa5)
